test/dotnet-new3.UnitTests/DotnetNewList.cs

```diff
@@ -60,5 +63,42 @@ public void CanFilterTags()
                 .And.NotHaveStdOutMatching("dotnet gitignore file\\s+gitignore\\s+Config")
                 .And.HaveStdOutMatching("Class Library\\s+classlib\\s+\\[C#\\],F#,VB\\s+Common/Library");
         }
+
+        [Fact]
+        public void CanSortByName()
+        {
+            const string expectedOutput =
+@"Template Name                                 Short Name     Language    Tags                  
```

Comment:
It is probably a good idea to set the `CultureInfo.CurrentUICulture` to `en-us` or `InvariantCulture` in the test class constructor so that these tests will still pass in machines with different locale.